src/navigation/NavBar.js

```diff
@@ -26,77 +25,73 @@ const NavLink = ({
-        </button>
+        <a href="#/welcome" role='menuitem' title="IPFS">
+          <div className='pt3 pb1 pv4-l'>
+            <img className='center db-l dn' style={{ height: 100 }} src={ipfsLogoTextVert} alt='IPFS' />
+            <img className='center db dn-l' style={{ height: 70 }} src={ipfsLogoTextHoriz} alt='IPFS' />
+          </div>
+        </a>
         <div className='db overflow-x-scroll overflow-x-hidden-l nowrap tc' role='menubar'>
-          <NavLink to='/' exact icon={StrokeMarketing} open={open}>{t('status:title')}</NavLink>
-          <NavLink to='/files' icon={StrokeWeb} open={open}>{t('files:title')}</NavLink>
-          <NavLink to='/explore' icon={StrokeIpld} open={open}>{t('explore:tabName')}</NavLink>
-          <NavLink to='/peers' icon={StrokeCube} open={open}>{t('peers:title')}</NavLink>
-          <NavLink to='/settings' icon={StrokeSettings} open={open}>{t('settings:title')}</NavLink>
+          <NavLink to='/' exact icon={StrokeMarketing}>{t('status:title')}</NavLink>
+          <NavLink to='/files' icon={StrokeWeb}>{t('files:title')}</NavLink>
+          <NavLink to='/explore' icon={StrokeIpld}>{t('explore:tabName')}</NavLink>
+          <NavLink to='/peers' icon={StrokeCube}>{t('peers:title')}</NavLink>
+          <NavLink to='/settings' icon={StrokeSettings}>{t('settings:title')}</NavLink>
         </div>
       </div>
-      { open &&
-        <div className='dn db-l navbar-footer mb3 center'>
-          { gitRevision && <div className='tc mb1'>
-            <a className='link white f7 o-80 glow' href={revisionUrl} target='_blank' rel='noopener noreferrer'>{t('status:revision')} {gitRevision}</a>
-          </div> }
-          <div className='flex flex-colum'>
-            <a className='link white f7 o-50 glow' href={codeUrl} target='_blank' rel='noopener noreferrer'>{t('status:codeLink')}</a>
-            <span className='mh2 white f7 o-50'>|</span>
-            <a className='link white f7 o-50 glow' href={bugsUrl} target='_blank' rel='noopener noreferrer'>{t('status:bugsLink')}</a>
-          </div>
+      <div className='dn db-l navbar-footer mb3 tc center f7 o-80 glow'>
+        { gitRevision && <div className='mb1'>
+          <a className='link white' href={revisionUrl} target='_blank' rel='noopener noreferrer'>{t('status:revision')} {gitRevision}</a>
         </div> }
+        <div className='mb1'>
+          <a className='link white' href={codeUrl} target='_blank' rel='noopener noreferrer'>{t('status:codeLink')}</a>
+        </div>
+        <div>
+          <a className='link white' href={bugsUrl} target='_blank' rel='noopener noreferrer'>{t('status:bugsLink')}</a>
+        </div>
+      </div>
     </div>
   )
 }
 
-export const NavBarContainer = ({ doToggleNavbar, navbarIsOpen, navbarWidth, ...props }) => {
+export const NavBarContainer = ({ ...props }) => {
```

Comment:
NavbarContainer used to handle the global state, now it's just pointless. I think you can remove it completely and just use Navbar in line 96

Reply:
@rafaelramalho19 indeed you can, thanks!